packages/components/pill/README.mdx

```diff
@@ -32,14 +32,6 @@ import { Pill } from '@contentful/f36-pill';
 
 ```
 
-### Integration example with `react-sortable-hoc`
```

Comment:
I guess you will provide a new example when the Pill changes PR is merged, right?